examples/cosmodc2/config.yml

```diff
@@ -1,16 +1,7 @@
 global:
     pixelization: healpix
     chunk_rows: 1000000
```

Comment:
Is this not getting defined twice?

Reply:
never mind one is `chunk_rows` and other is `chunk_size`, why do we have both?

Reply:
The RAIL people use `chunk_size` and TXPipe uses `chunk_rows`. At some point we should probably just update TXPipe to copy RAIL, there's fewer repos to change for us.